huxley/www/js/stores/__tests__/DelegateStore-test.js

```diff
@@ -40,4 +42,41 @@ describe('DelegateStore', () => {
       }),
     ]);
   });
+
+  it('adds a delegate', () => {
+    return Promise.all([
+      DelegateStore.getDelegates(mockSchoolId, (delegates) => {
+        expect(delegates).toEqual(mockDelegates);
+        DelegateStore.addDelegate({id: 3, name: 'Trevor', email: ''});
+        expect(DelegateStore.getDelegates(mockSchoolId).length).toEqual(3);
+      })
+    ]);
+  });
+
+  it('deletes a delegate', () => {
+    return Promise.all([
+      DelegateStore.getDelegates(mockSchoolId, (delegates) => {
+        expect(delegates).toEqual(mockDelegates);
+        DelegateStore.deleteDelegate(mockDelegates[0]);
+        expect(DelegateStore.getDelegates(mockSchoolId).length).toEqual(1);
+      })
+    ]);
+  });
+
+  it('updates a delegate', () => {
+    return Promise.all([
+      DelegateStore.getDelegates(mockSchoolId, (delegates) => {
+        expect(delegates).toEqual(mockDelegates);
+        var updatedDelegate = mockDelegates[0]
```

Comment:
I don't think this test works as expected, because `delegates` and `mockDelegates` are literally the same object. That means, if you update one delegate's name in-place (like below), it already updates it in the store (which is no bueno).

Instead, just write out the new name in the `updateDelegate` call:
```js
DelegateStore.updateDelegate(
  d.id,
  'Jake Moskowitz',
  d.email,
  mockSchoolId,
);
```